index.html

```diff
@@ -1,28 +1,34 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>My Portfolio</title>
-  <link rel="stylesheet" href="style.css">
-</head>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
 <body>
-  <nav class="navbar">
-    <h1>MY Logo</h1>
-    <ul class="nav-menu">
-      <li><a href="#">Home</a></li>
-      <li><a href="#">About</a></li>
-      <li><a href="#">Contact</a></li>
-    </ul>
-    <img  class="hamburger" src="assets/menu.png" width="50px"  alt="Logo" />
+  <header>
+    <nav class="navbar">
+      <h1 ><a class="my-logo" href="#section2">MY Logo </a></h1>
```

Comment:
Great job in wrapping this section with a <code>header</code> tag and making your logo a link 👌👌👌